index.js

```diff
@@ -74,43 +74,6 @@ Cu.import("resource:///modules/sessionstore/SessionStore.jsm");
 Cu.import("resource://gre/modules/Services.jsm");
 
 // ----------------------------------------------------------------------------
-// ContextualIdentityProxy
-
-const ContextualIdentityProxy = {
```

Comment:
This needs Firefox 54 for removal... we also need to change it to the "Public" method instead.

The upgrade code in the install process doesn't seem to work on >54 so I need to make it support both.